cmd/statusd-cli/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"net"
+	"os"
+)
+
+const (
+	// StatusDHost is the default host to connect to.
+	StatusDHost = "localhost"
+
+	// StatusDPort is the default port to connect to.
+	StatusDPort = "51515"
+)
+
+var (
+	statusdHost = flag.String("statusd", StatusDHost, "set host of statusd connection")
```

Comment:
For simplicity, you could write:
```go
statusdAddr = flag.String("statusdaddr", StatusdAddr, "set statusd address (default localhost:51515)")
```

This is ok as well.

Reply:
Hehe, this had been the beginning. It then has been changed to one standard port, then an optional but standardised port, and now will go back to an obligatory port.